@xen-orchestra/xapi/vm.mjs

```diff
@@ -564,10 +563,10 @@ class Vm {
       )
     }
     try {
-      const ref = await this.putResource(stream, '/import/', {
+      const [ref] = await this.putResource(stream, '/import/', {
```

Comment:
Why the return of the `putResource` become an array?

Reply:
Because that's the return value of an import.

IIRC, an XVA can contain multiple VMs.